db/migrate/20160531152256_add_payment_mode_to_conferences.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentModeToConferences < ActiveRecord::Migration
+  def change
+    add_column :conferences, :payment_mode, :string, default: 'Offline'
```

Comment:
Also, how about using Enums?
I think it fits this use case pretty nicely.
http://api.rubyonrails.org/v4.1.0/classes/ActiveRecord/Enum.html